src/slider.jsx

```diff
@@ -199,7 +193,7 @@ const Slider = React.createClass({
   getStyles() {
     const fillGutter = this.getTheme().handleSize / 2;
     const disabledGutter = this.getTheme().trackSize + this.getTheme().handleSizeDisabled / 2;
-    const calcDisabledSpacing = this.props.disabled ? ` -${disabledGutter}px` : '';
+    const calcDisabledSpacing = this.props.disabled ? ` - ${disabledGutter}px` : '';
```

Comment:
I wonder, when this is not disabled, does it matter if it's `undefined` vs `''`. If not, we could write this as
```
this.props.disabled && `-{disabledGutter}px`
```